addons/mail/static/src/js/mail.js

```diff
@@ -855,7 +857,7 @@ var MailThread = Attachment.extend ({
 
         this.followers_loaded = $.Deferred();
 
-        if ((! this.nb_followers || !this.follower_ids) && (this.model && this.res_id)) {
+        if ((! this.nb_followers || !this.follower_ids) && (this.model && this.res_id) && this.options.view_inbox) {
```

Comment:
Is this related to the tips ? If not, it should go in a different commit in another branch.